schema.json

```diff
@@ -456,17 +456,18 @@
             "description": "Big type of layer. Currently all raster layers are wms layers and Vector layers are subdivided by protocol property",
             "default": "",
             "enum": [
-              "HSLayers.Layer.WMS",
-              "OpenLayers.Layer.Vector",
+              "WMS",
               "WMS",
               "WFS",
```

Comment:
"ClassName: WFS" is never used, I suggest to remove it also.

Reply:
Not used in HSLayers either. But wouldn't it be future-proof to keep the WFS classname as an allowed value?